script_mteb_french/results_analysis/all_pearson_correlations.json

```diff
@@ -0,0 +1,18 @@
+{
+    "bleu": {
+        "vs_human": 0.6932073280651037,
+        "vs_text": 0.9083528795712384
+    },
+    "rouge1": {
+        "vs_human": 0.8491076070563548,
+        "vs_text": 0.4206552621087066
```

Comment:
What's `vs_text` ? human summaries in French?

Reply:
In the datasets, you have human summaries, machine summaries and text. For both French and English, I compute all the scores between machine summaries and human summaries (vs_human scores) and all the scores between machine summaries and text (vs_text scores): https://github.com/Lyon-NLP/mtebscripts/pull/116/files#diff-9943f6a7700a81d1440b9cdaf08bbd7913c108f877c7f74045a9fd5f0c2543fdR97-R114

And then I compute all correlations between English and French. 'vs_human' means the correlations of the score "machine vs human" between English and French.

Reply:
Since the evaluation of the summarisation task is based only on human and machine summaries, I'll only report the vs_human scores in the paper.